source/common/http/match_delegate/config.cc

```diff
@@ -327,10 +330,14 @@ FilterConfigPerRoute::createFilterMatchTree(
       std::make_unique<envoy::extensions::filters::common::dependency::v3::MatchingRequirements>();
   requirements->mutable_data_input_allow_list()->add_type_url(TypeUtil::descriptorFullNameToTypeUrl(
       envoy::type::matcher::v3::HttpRequestHeaderMatchInput::descriptor()->full_name()));
+  requirements->mutable_data_input_allow_list()->add_type_url(TypeUtil::descriptorFullNameToTypeUrl(
```

Comment:
@tyxia Is this what you meant by add in the requirements.

Reply:
Yes.